card/src/main/java/com/adyen/checkout/card/internal/ui/view/CardView.kt

```diff
@@ -425,6 +429,15 @@ class CardView @JvmOverloads constructor(
                 binding.textInputLayoutExpiryDate.showError(localizedContext.getString(expiryDateValidation.reason))
             }
         }
+        binding.editTextExpiryDate.setContentDescription()
+    }
+
+    // TODO - Investigate contentDescription in ViewFieldState [COSDK-490]
+    private fun ExpiryDateInput.setContentDescription() {
```

Comment:
`setContentDescription` doesn't really apply anymore now, maybe we can rename it to something like `setAccessibilityDescription`? Also for the method on line 461